dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3TreeTablePresenter.java

```diff
@@ -729,4 +706,44 @@ private void handleDragDetectedEvent(final Event event) {
         event.consume();
     }
 
+    private void startPutJob(final Session session,
+            final List<Pair<String, Path>> files,
+            final String bucket,
+            final String targetDir,
+            final JobInterruptionStore jobInterruptionStore) {
+        final Ds3Client client = session.getClient();
+
+        final ImmutableList.Builder<kotlin.Pair<String,Path>> builder = ImmutableList.builder();
+        files.forEach(f -> builder.add(new kotlin.Pair<>(f.getKey(), f.getValue())));
```

Comment:
Can we make f file?  When I'm reading f.getWhatever(), I have to seek back to the beginning of the line to what f is.